webknossos/webknossos/dataset/dataset.py

```diff
@@ -1289,7 +1289,7 @@ def add_layer_from_images(
                 set(layer.bounding_box.axes).difference("x", "y", "z")
             ) and layer.data_format != DataFormat.Zarr3:
```

Comment:
`DataFormat.Zarr` should also be fine, no?

Reply:
It didn't work before, but I fixed the issue and changed this part in code, to allow Zarr and Zarr3.